app/screens/sso/sso.js

```diff
@@ -235,7 +239,7 @@ class SSO extends PureComponent {
                     onNavigationStateChange={this.onNavigationStateChange}
                     onShouldStartLoadWithRequest={() => true}
                     renderLoading={this.renderLoading}
-                    onMessage={this.onMessage}
+                    onMessage={this.state.onMessage}
```

Comment:
Passing function to `state.onMessage` looks straight forward but it's not obvious to me if there's error or not. What I'm thinking is something like: 
```
onMessage={this.state.error ? null : this.onMessage}
```

Reply:
Ok is not exacly looking for an error but I see your point and I can use that